packages/examples/README.md

```diff
@@ -0,0 +1,13 @@
+# VA Design System Integration Examples
+
+This directory contains examples of how to integrate the VA Design System into different frameworks and build tools outside of vets-website ([VA.gov](https://VA.gov)).
+
+## Available Examples
+
+- [Vite + React + TypeScript](./vite-react-typescript)
+- [CodeSandbox](https://codesandbox.io/p/sandbox/suspicious-stonebraker-vzfzhw)
+- [CodePen](https://codepen.io/jamigibbs-the-sans/pen/jEEdOmY)
```

Comment:
I could not get the icon sprite sheet to load in the CodePen example because the file needs to be added to an `/img` folder within the project. CodePen allows you to upload assets with a pro account but not with the free service.

Reply:
Can we provide the sprite sheet separately with instructions specifically for CodePen? CodePen is a popular option for designers currently.

Reply:
Linking to an issue created about adding documentation for hosting the sprite sheet:

- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4288